pkg/core/fixtures/simple/.traefik.yml

```diff
@@ -0,0 +1,7 @@
+displayName: Plugin With Unsafe
+type: middleware
+
+import: github.com/traefik/plugintestunsafe
+basePkg: unsafe
+
+summary: Simple example plugin with unsafe.
```

Comment:
As we have an `unsafe` dedicated fixture, I suppose this one is meant to be just a simple example plugin.